packages/rancher-backup/package.yaml

```diff
@@ -1,6 +1,6 @@
 url: local
 packageVersion: 01
-releaseCandidateVersion: 00
+releaseCandidateVersion: 01
```

Comment:
Assuming this is just to fix CI, if its still a problem we need to look into it. 
I don't see backup on our list of new charts in 2.5.6, can you add it there.

Reply:
https://github.com/rancher/charts/pull/935 << yeah it looks like this pr was merged without an rc change.

Reply:
Yup, this looks right. The sync workflow (applied on a push to dev-v2.5-source) also seems to have failed accordingly, so the changes from that PR never made it into dev-v2.5: https://github.com/rancher/charts/runs/1875711331

Seems like the push from this PR took care of that though, as seen here:
https://github.com/rancher/charts/commit/fc035102ae98090412de5b33615f6d71c866a741

Reply:
I rebased with my current PR and the CI passed. Think that did the trick
https://github.com/rancher/charts/pull/991